rasa_sdk/knowledge_base/actions.py

```diff
@@ -205,21 +224,47 @@ def _query_attribute(
             dispatcher.utter_message(template="utter_ask_rephrase")
             return [SlotSet(SLOT_MENTION, None)]
 
-        object_of_interest = self.knowledge_base.get_object(object_type, object_name)
+        if utils.is_coroutine_action(self.knowledge_base.get_object):
+            object_of_interest = await self.knowledge_base.get_object(
```

Comment:
should be async in the interface

Reply:
please also update `InMemoryKnowledgeBase`